packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.tsx

```diff
@@ -255,12 +256,15 @@ export const YppAuthorizationModal: FC<YppAuthorizationModalProps> = ({ unSynced
         },
         onTxSync: async ({ channelId }) => {
           setActiveChannel(channelId)
+          createdChannelId.current = channelId
         },
         onCompleted: async () => {
           await refetchUserMemberships()
 
           const channelCreationResponse = await yppSignChannelMutation({
-            ...(selectedChannelId ? { joystreamChannelId: parseInt(selectedChannelId) } : {}),
+            ...(selectedChannelId || createdChannelId.current
+              ? { joystreamChannelId: parseInt(selectedChannelId || createdChannelId.current || '') }
```

Comment:
`parseInt` from `''` will return `NaN`

Reply:
@WRadoslaw thats true, but then it won't pass the ternary condition above? `selectedChannelId || createdChannelId.current` either of them should be truthy value

Reply:
anyways, I've put it into a separate value to make the code look cleaner